### DISCUSSION TITLE
Support Georgian locale

### DISCUSSION THREAD

**Discussion by yaroslavnabokov** [adobe/react-spectrum#7257] [General]
Hi, i'm working on adding localization for website. Unfortunately, Georgian is not in the list of supported languages. Are you planning to add localization for this language in the near future?

I would like to make one observation: although the language is not supported on the client, the server returns the translated text. Perhaps I can make a crutch as a temporary solution?

Thank you!

**Comment by snowystinger** [comment]
Linking https://github.com/adobe/react-spectrum/issues/9672
